docker/gcp_pudl_etl.sh

```diff
@@ -79,23 +79,54 @@ function upload_to_dist_path() {
     GCS_PATH="gs://pudl.catalyst.coop/$1/"
     AWS_PATH="s3://pudl.catalyst.coop/$1/"
 
-    # If the old outputs don't exist, these will exit with status 1, so we
-    # don't && them with the rest of the commands.
-    echo "Removing old outputs from $GCS_PATH."
-    gsutil -m -u "$GCP_BILLING_PROJECT" rm -r "$GCS_PATH"
-    echo "Removing old outputs from $AWS_PATH."
-    aws s3 rm "$AWS_PATH" --recursive
-
-    echo "Copying outputs to $GCS_PATH:" && \
-    gsutil -m -u "$GCP_BILLING_PROJECT" cp -r "$PUDL_OUTPUT/*" "$GCS_PATH" && \
-    echo "Copying outputs to $AWS_PATH" && \
-    aws s3 cp "$PUDL_OUTPUT/" "$AWS_PATH" --recursive
+    # Only attempt to update outputs if we have an argument
+    # This avoids accidentally blowing away the whole bucket if it's not set.
+    if [[ -n "$1" ]]; then
```

Comment:
I think if we called this function without an argument it would delete all of our previous releases which is bad, so I added this small margin of safety.  Maybe we should automatically make the versioned releases read-only as soon as we create them so this can't happen.

`stable` should always point at a versioned release, so replacing it if need be wouldn't be hard.  `nightly` is ephemeral so not a disaster if it accidentally gets wiped out. And the build outputs for recent `nightly` builds are still laying around so we could repopulate it manually.

Reply:
Great. Hopefully this is easy?  I'd like to add this to the process / script for our next release.